### PR TITLE
tiup: add kvcdc server topylogy example

### PR DESCRIPTION
**Pull request by wk989898**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

topology config supports TiCDC server and TiKV-CDC, they are different.
https://asktug.com/t/topic/1034131
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by flowbehappy — APPROVED** [review]

File: tiup/tiup-cluster-topology-reference.md [hunk 1]
```diff
@@ -463,6 +463,50 @@ tiproxy_servers:
   - host: 10.0.1.22
 ```
 
+### `kvcdc_servers`
+
+`kvcdc_servers` specifies the machines to which the TiKV-CDC services are deployed. It also specifies the service configuration on each machine. `kvcdc_servers` is an array. Each array element contains the following fields:
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:468]:
```suggestion
`kvcdc_servers` specifies the machines to which the [TiKV-CDC](https://tikv.org/docs/7.1/concepts/explore-tikv-features/cdc/cdc/) services are deployed. It also specifies the service configuration on each machine. `kvcdc_servers` is an array. Each array element contains the following fields:
```

File: tiup/tiup-cluster-topology-reference.md [hunk 2]
```diff
@@ -463,6 +463,50 @@ tiproxy_servers:
...
+
+- `addr`: The listening address of TiKV-CDC, the HTTP API address, and the Prometheus address of the TiKV-CDC service. The default value is 127.0.0.1:8600.
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:470]:
```suggestion
- `addr`: The listening address of TiKV-CDC, the HTTP API address, and the Prometheus address of the TiKV-CDC service. The default value is `127.0.0.1:8600`.
```

File: tiup/tiup-cluster-topology-reference.md [hunk 3]
```diff
@@ -463,6 +463,50 @@ tiproxy_servers:
...
+
+- `advertise-addr`: The advertised address via which clients access TiKV-CDC. If unspecified, the value is the same as addr.
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:472]:
```suggestion
- `advertise-addr`: The advertised address via which clients access TiKV-CDC. If not specified, the value is the same as that of `addr`.
```

File: tiup/tiup-cluster-topology-reference.md [hunk 4]
```diff
@@ -463,6 +463,50 @@ tiproxy_servers:
...
+
+- `pd`: A comma-separated list of PD endpoints.
+
+- `config`: The address of the configuration file that TiKV-CDC uses (optional).
+
+- `data-dir`: Specifies the directory that TiKV-CDC uses to store temporary files for sorting. It is recommended to ensure that the free disk space for this directory is greater than or equal to 500 GiB (optional).
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:478]:
```suggestion
- `data-dir`: The directory that TiKV-CDC uses to store temporary files for sorting. It is recommended to ensure that the free disk space for this directory is greater than or equal to 500 GiB (optional).
```

File: tiup/tiup-cluster-topology-reference.md [hunk 5]
```diff
@@ -463,6 +463,50 @@ tiproxy_servers:
...
+
+- `gc-ttl`: The TTL (Time To Live, in seconds) of the service level GC safepoint in PD set by TiKV-CDC (optional). It's the duration that replication tasks can suspend, defaults to 86400, that is 24 hours. Note that suspending of replication task will affect the progress of TiKV garbage collection. The longer of gc-ttl, the longer a changefeed can be paused, but at the same time more obsolete data will be kept and larger space will be occupied. Vice versa.
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:480]:
```suggestion
- `gc-ttl`: The TTL (Time to Live, in seconds) of the service-level GC safepoint in PD set by TiKV-CDC (optional). It is the duration that replication tasks can be suspended, defaulting to `86400`, which is 24 hours. Note that suspending replication tasks affects the progress of TiKV garbage collection. The longer the `gc-ttl`, the longer changefeeds can be paused, but at the same time, more obsolete data will be kept and occupy more space. Vice versa.
```

File: tiup/tiup-cluster-topology-reference.md [hunk 6]
```diff
@@ -463,6 +463,50 @@ tiproxy_servers:
...
+
+- `log-file`: The path to which logs are output when the TiKV-CDC process is running (optional). If this parameter is not specified, logs are written to the standard output (stdout).
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:482]:
```suggestion
- `log-file`: The path to which logs are output when the TiKV-CDC process is running (optional). If not specified, logs are written to the standard output (stdout).
```

File: tiup/tiup-cluster-topology-reference.md [hunk 7]
```diff
@@ -463,6 +463,50 @@ tiproxy_servers:
...
+
+- `log-level`: The log level when the TiKV-CDC process is running (optional). The default value is "info".
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:484]:
```suggestion
- `log-level`: The log level when the TiKV-CDC process is running (optional). The default value is `"info"`.
```

File: tiup/tiup-cluster-topology-reference.md [hunk 8]
```diff
@@ -463,6 +463,50 @@ tiproxy_servers:
...
+
+- `ca`: The path of the CA certificate file in PEM format for TLS connection (optional).
+
+- `cert`: The path of the certificate file in PEM format for TLS connection (optional).
+
+- `key`: The path of the private key file in PEM format for TLS connection (optional).
+
+- `cert-allowed-cn`: Specifies to verify caller's identity (certificate Common Name, optional). Use comma to separate multiple CN.
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:492]:
```suggestion
- `cert-allowed-cn`: Controls whether to verify the caller's identity (certificate Common Name, optional). Use commas to separate multiple CNs.
```

File: tiup/tiup-cluster-topology-reference.md [hunk 9]
```diff
@@ -463,6 +463,50 @@ tiproxy_servers:
...
+
+Among the above fields, you cannot modify the following configured fields after the deployment:
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:494]:
```suggestion
For the above fields, you cannot modify these configured fields after the deployment:
```

**Review by qiancai — APPROVED** [review]

File: tiup/tiup-cluster-topology-reference.md [hunk 10]
```diff
@@ -463,6 +464,50 @@ tiproxy_servers:
...
+
+- `log-level`: The log level when the TiKV-CDC process is running (optional). The default value is `"info"`.
+
+- `ca`: The path of the CA certificate file in PEM format for TLS connection (optional).
+
+- `cert`: The path of the certificate file in PEM format for TLS connection (optional).
+
+- `key`: The path of the private key file in PEM format for TLS connection (optional).
+
+- `cert-allowed-cn`: Controls whether to verify the caller's identity (certificate Common Name, optional). Use commas to separate multiple CNs.
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:493]:
```suggestion
- `cert-allowed-cn`: The allowed caller identities (certificate Common Name, optional). Use commas to separate multiple CNs.
```

File: tiup/tiup-cluster-topology-reference.md [hunk 11]
```diff
@@ -463,6 +464,50 @@ tiproxy_servers:
   - host: 10.0.1.22
 ```
 
+### `kvcdc_servers`
+
+`kvcdc_servers` specifies the machines to which the [TiKV-CDC](https://tikv.org/docs/7.1/concepts/explore-tikv-features/cdc/cdc/) services are deployed. It also specifies the service configuration on each machine. `kvcdc_servers` is an array. Each array element contains the following fields:
+
+- `addr`: The listening address of TiKV-CDC, the HTTP API address, and the Prometheus address of the TiKV-CDC service. The default value is `127.0.0.1:8600`.
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:471]:
```suggestion
- `addr`: The listening address of TiKV-CDC, used to provide HTTP API and Prometheus queries. The default value is `127.0.0.1:8600`.
```

File: tiup/tiup-cluster-topology-reference.md [hunk 12]
```diff
@@ -463,6 +464,50 @@ tiproxy_servers:
...
+
+- `addr`: The listening address of TiKV-CDC, used to provide HTTP API and Prometheus queries. The default value is `127.0.0.1:8600`.
+
+- `advertise-addr`: The advertised address via which clients access TiKV-CDC. If not specified, the value is the same as that of `addr`.
+
+- `pd`: A comma-separated list of PD endpoints.
+
+- `config`: The address of the configuration file that TiKV-CDC uses (optional).
+
+- `data-dir`: The directory that TiKV-CDC uses to store temporary files for sorting. It is recommended to ensure that the free disk space for this directory is greater than or equal to 500 GiB (optional).
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:479]:
```suggestion
- `data-dir`: The directory that TiKV-CDC uses to store temporary files mainly for sorting. It is recommended to ensure that the free disk space for this directory is greater than or equal to 500 GiB (optional).
```

File: tiup/tiup-cluster-topology-reference.md [hunk 13]
```diff
@@ -463,6 +464,50 @@ tiproxy_servers:
...
+
+- `gc-ttl`: The TTL (Time to Live, in seconds) of the service-level GC safepoint in PD set by TiKV-CDC (optional). It is the duration that replication tasks can be suspended, defaulting to `86400`, which is 24 hours. Note that suspending replication tasks affects the progress of TiKV garbage collection. The longer the `gc-ttl`, the longer changefeeds can be paused, but at the same time, more obsolete data will be kept and occupy more space. Vice versa.
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:481]:
```suggestion
- `gc-ttl`: The TTL (Time to Live, in seconds) of the service-level GC safepoint in PD set by TiKV-CDC (optional). It is the duration that replication tasks can be suspended, defaulting to `86400`, which is 24 hours. Note that suspending replication task affects the progress of TiKV garbage collection. The longer the `gc-ttl`, the longer changefeeds can be paused, but at the same time, more obsolete data will be kept and occupy more space. Vice versa.
```

File: tiup/tiup-cluster-topology-reference.md [hunk 14]
```diff
@@ -463,6 +464,50 @@ tiproxy_servers:
...
+
+- `ca`: The path of the CA certificate file in PEM format for TLS connection (optional).
+
+- `cert`: The path of the certificate file in PEM format for TLS connection (optional).
+
+- `key`: The path of the private key file in PEM format for TLS connection (optional).
+
+- `cert-allowed-cn`: The allowed caller identities (certificate Common Name, optional). Use commas to separate multiple CNs.
+
+For the above fields, you cannot modify these configured fields after the deployment:
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:495]: Sync the following with the Chinese version if updated.

**Review by xhebox — APPROVED** [review]

File: tiup/tiup-cluster-topology-reference.md (hunk 14 above)
Comment (qiancai) [reply tiup/tiup-cluster-topology-reference.md:497]: done in [`3714c46` (#19230)](https://github.com/pingcap/docs/pull/19230/commits/3714c4678533c4654c7fdd77c47dfa4d654b8c7e)

File: tiup/tiup-cluster-topology-reference.md [hunk 15]
```diff
@@ -463,6 +464,54 @@ tiproxy_servers:
...
+
+- `host`: Specifies the machine to which the TiKV-CDC services are deployed. The field value is an IP address and is mandatory.
+
+- `ssh_port`: Specifies the SSH port to connect to the target machine for operations. If it is not specified, the `ssh_port` of the `global` section is used.
+
+- `port`: The listening port of the TiKV-CDC services. The default value is `8600`.
+
+- `deploy_dir`: Specifies the deployment directory. If it is not specified or specified as a relative directory, the directory is generated according to the `deploy_dir` directory configured in `global`.
+
+- `data-dir`: The directory that TiKV-CDC uses to store temporary files primarily for sorting. It is recommended to ensure that the free disk space for this directory is greater than or equal to 500 GiB (optional).
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:479]:
```suggestion
- `data-dir`: Specifies the directory that TiKV-CDC uses to store temporary files primarily for sorting (optional). The free disk space for this directory is recommended to be greater than or equal to 500 GiB.
```

**Review by hfxsd — APPROVED** [review]

File: tiup/tiup-cluster-topology-reference.md [hunk 16]
```diff
@@ -463,6 +464,54 @@ tiproxy_servers:
...
+
+- `port`: The listening port of the TiKV-CDC services. The default value is `8600`.
+
+- `deploy_dir`: Specifies the deployment directory. If it is not specified or specified as a relative directory, the directory is generated according to the `deploy_dir` directory configured in `global`.
+
+- `data-dir`: Specifies the directory that TiKV-CDC uses to store temporary files primarily for sorting (optional). The free disk space for this directory is recommended to be greater than or equal to 500 GiB.
+
+- `log_dir`: Specifies the log directory. If it is not specified or specified as a relative directory, the log is generated according to the `log_dir` directory configured in `global`.
+
+- `gc-ttl`: The TTL (Time to Live, in seconds) of the service-level GC safepoint in PD set by TiKV-CDC (optional). It is the duration that replication tasks can be suspended, defaulting to `86400`, which is 24 hours. Note that suspending replication tasks affect the progress of TiKV garbage collection safepoint. The longer the `gc-ttl`, the longer changefeeds can be suspended, but at the same time, more obsolete data will be kept and occupy more space. Vice versa.
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:483]:
```suggestion
- `gc-ttl`: The TTL (Time to Live, in seconds) of the service-level GC safepoint in PD set by TiKV-CDC (optional). It is the duration that replication tasks can be suspended, defaulting to `86400`, which is 24 hours. Note that suspending replication tasks affects the progress of TiKV garbage collection safepoint. The longer the `gc-ttl`, the longer changefeeds can be suspended, but at the same time, more obsolete data will be kept and occupy more space. Vice versa.
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**